lib/src/__fixtures__/nanotdf-spec-plain-embedded-example.ts

```diff
@@ -28,7 +28,31 @@ LWExNjctZWY5OGQyY2NmNjdhIn2avSz7nTV08u+z0lNoOax2ZSWlNtycmvQLS4zHNJn/2i8E1p+KGUx+
 eO9leyauC42VKckiAm5GJxs=
 `;
 
-export const header = {
+export interface Header {
+  magicNumberVersion: string[];
+  kas: {
+    protocol: number;
+    length: number;
+    body: string;
+  };
+  eccBindingMode: {
+    useECDSABinding: boolean;
+    ephemeralCurveName: number;
+  };
+  symmetricPayloadConfig: {
```

Comment:
we could make it as interface also and reuse

Reply:
same as above